code/datums/spells/lightning.dm

```diff
@@ -73,17 +75,33 @@ obj/effect/proc_holder/spell/targeted/lightning/proc/Reset(mob/user = usr)
 	user.Beam(target,icon_state="lightning[rand(1,12)]",icon='icons/effects/effects.dmi',time=5)
 
 	var/energy = min(world.time - start_time,100)
-	Bolt(user,target,max(15,energy/2),5,user) //5 bounces for energy/2 burn
+	if(damaging)
+		Bolt(user,target,max(15,energy/2),5,user) //5 bounces for energy/2 burn
+	else
+		var/bounces = round(energy/20)
+		Bolt(user,target,0,bounces,user)
 	Reset(user)
 
-/obj/effect/proc_holder/spell/targeted/lightning/proc/Bolt(mob/origin, mob/target, bolt_energy, bounces, mob/user = usr)
+/obj/effect/proc_holder/spell/targeted/lightning/proc/Bolt(mob/origin, mob/living/target, bolt_energy, bounces, mob/user = usr)
 	origin.Beam(target,icon_state="lightning[rand(1,12)]",icon='icons/effects/effects.dmi',time=5)
-	var/mob/living/carbon/current = target
+	var/mob/living/current = target
 	if(bounces < 1)
-		current.electrocute_act(bolt_energy,"Lightning Bolt",safety=1)
```

Comment:
Except the falling over itself is OP since it makes you drop items. Since this is an AoE for a non-antag RP character, I figured just the stun+jitteriness would be annoying yet not too critical.